clippy_tests/examples/doc.stderr

```diff
@@ -206,6 +206,14 @@ error: you should put `be_sure_we_got_to_the_end_of_it` between ticks in the doc
     |
     = note: `-D doc-markdown` implied by `-D warnings`
 
+error: you should put `http://www.unicode.org/reports/tr9/#Reordering_Resolved_Levels` between ticks in the documentation
```

Comment:
Done. I meant for the first part, for the "lint that detects code which applies a heuristic to text to find urls and suggests to use the `url` crate instead" we'll see in another PR 😆